src/opentimelineio/multiMediaReference.cpp

```diff
@@ -1,23 +1,22 @@
-#include "opentimelineio/mediaReference.h"
+#include "opentimelineio/multiMediaReference.h"
 
 namespace opentimelineio { namespace OPENTIMELINEIO_VERSION {
 
 MultiMediaReference::MultiMediaReference(
     std::string const& name, AnyDictionary const& metadata)
     : Parent(name, metadata)
-    , _available_range(available_range)
 {}
 
 MultiMediaReference::~MultiMediaReference()
 {}
 
 optional<TimeRange>
-MultiMediaReference::available_range() const noexcept final
+MultiMediaReference::available_range() const noexcept
 {}
```

Comment:
Isn't this method body missing a return statement?

Reply:
I think this whole file can just be removed! Media Multi-Reference is handled at the clip level.  It looks like was an early prototype I was working on that accidentally got checked in with my spatial coordinates branch in commit 8f93157e.  Sorry about that!

Reply:
It's odd that this file was being compiled, right? It's not in the CMakeFile ...

Reply:
yes. Now that you mention it, I believe the intent was that these functions don't do anything. Weird that the compiler doesn't flag it.

Reply:
@rogernelson The Swift Bindings compile all the cpp files. That explains why it's never shown up as an error before, since I have updated the Swift Bindings to latest.

Reply:
Oh, it's mean to be removed.